eis_toolkit/cli.py

```diff
@@ -3065,7 +3074,149 @@ def gamma_overlay_cli(input_rasters: INPUT_FILES_ARGUMENT, output_raster: OUTPUT
+
+    typer.echo("Progress: 10%")
+
+    # Each set of rasters contains either W+ and S_W+ or their generalized counterparts
+    n_raster_sets = int(len(input_rasters) / 2)
+    dict_array = [{} for _ in range(n_raster_sets)]
+
+    raster_profile = None
+
+    for raster_file in input_rasters:
+        file_name = os.path.splitext(os.path.basename(raster_file))[0]
+
+        with rasterio.open(raster_file) as raster:
+            if raster_profile is None:
+                raster_profile = raster.profile
+
+            array = raster.read(1)
+
+            # Assign array to the first dictionary that lacks its key
+            for dictionary in dict_array:
+                if file_name not in dictionary:
+                    dictionary[file_name] = array
+                    break
+
+    # Retain only the common keys
+    common_keys = set.intersection(*map(set, dict_array))
+    dict_array = [{key: dictionary[key] for key in common_keys} for dictionary in dict_array]
+    typer.echo(dict_array)
+
+    ######################################################################################################
+    # Another way would be to rely on raster file paths but I'm not sure if it would work with the plugin.
+    ######################################################################################################
+    # raster_dict = {}
+    # for raster_file in input_rasters:
+    #     folder = os.path.dirname(raster_file).split("/")[-1]
+    #     file = os.path.basename(raster_file)
+    #     file_name = os.path.splitext(file)[0]
+
+    #     with rasterio.open(raster_file) as raster:
+    #         if raster_profile is None:
+    #             raster_profile = raster.profile
+
+    #         array = raster.read(1)
+
+    #         if folder not in raster_dict:
+    #             raster_dict[folder] = {}
+
+    #         raster_dict[folder][file_name] = array
+
+    # dict_array = [raster_dict[folder] for folder in raster_dict]
+
+    typer.echo("Progress: 25%")
+
+    posterior_probabilities, posterior_probabilies_std, confidence_array = weights_of_evidence_calculate_responses(
+        output_arrays=dict_array, nr_of_deposits=nr_of_deposits, nr_of_pixels=nr_of_pixels
+    )
+    typer.echo("Progress: 75%")
+
+    output_probabilities_path = output_dir.joinpath("posterior_probabilities.tif")
+    output_probabilities_std_path = output_dir.joinpath("posterior_probabilities_std.tif")
+    output_confidence_array_path = output_dir.joinpath("confidence_array.tif")
+
+    with rasterio.open(output_probabilities_path, "w", **raster_profile) as dst:
+        dst.write(posterior_probabilities, 1)
+
+    with rasterio.open(output_probabilities_std_path, "w", **raster_profile) as dst:
+        dst.write(posterior_probabilies_std, 1)
+
+    with rasterio.open(output_confidence_array_path, "w", **raster_profile) as dst:
+        dst.write(confidence_array, 1)
+
+    typer.echo("Progress: 100%")
+
+    typer.echo(f"Results saved in {output_dir}.")
```

Comment:
```suggestion
    typer.echo(f"Responses calculations finished, writing output rasters to {output_raster_probabilities},  {output_raster_standard_deviations} and {output_raster_confidence}.")
```
If you change the output dir to output raster paths